gitlab-mixin/dashboards/gitlab-overview.libsonnet

```
@@ -680,6 +681,18 @@ local buildTraceOperationsPanel = {
             label='Instance',
             refresh='time'
           ),
+          template.new(
+            'cluster',
+            promDatasource,
+            'label_values(gitlab_rails_boot_time_seconds{job=~"$job"}, cluster)',
+            label='Cluster',
+            refresh=1,
```

This is the lint error. :)

```suggestion
            refresh=2,
```